site/_data/authors.yaml

```
@@ -0,0 +1,6 @@
+- name: Marcus Ilgner
+  github: milgner
+  web: https://illunis.net
```

I'll have to update my website soon :) We can put `https://marcusilgner.com` here.
```suggestion
  web: https://marcusilgner.com
```